projects/defibox/index.js

```diff
@@ -0,0 +1,41 @@
+const axios = require("axios");
+const utils = require("../helper/utils");
+
+const eosEndpoint = "https://dapp.defibox.io/api/"
+const bscEndpoint = "https://bsc.defibox.io/api/"
+
+async function eos() {
+  const eosPrice = (await utils.getPricesfromString("eos")).data.eos.usd;
+  const swap = await utils.fetchURL(eosEndpoint + "swap/get24HInfo")
+  const lend = await utils.fetchURL(eosEndpoint + "lend/getGlobalOpenPositionStat")
+  const usn = await utils.fetchURL(eosEndpoint + "st/open/getGlobalOpenStat")
+  const tvl = Number(swap.data.data.eosBalance) * eosPrice * 2 +
+              lend.data.data.practicalBalance +
```

Comment:
Is the lending tvl equal to the assets on the smart contract or the total of assets that have been deposited (including borrowed assets)

Reply:
Is there a way we could calculate the amount excluding borrrowed assets? That's what we are doing for other lending protocols so it would make sense to do the same here to keep comparisons fair

Reply:
@0xngmi sent a new PR to update Defibox lend methodology

https://github.com/DefiLlama/DefiLlama-Adapters/pull/442

Now deducts borrowed assets from TVL

https://defibox.io/lend

```
Total deposit
39,299,866 USD

Total borrow
4,601,793 USD

Total TVL
34,698,073 USD
```

Defibox DefiLlama TVL:

```
--- eos ---
Total: 137.88 M
--- bsc ---
Total: 6.74 M
--- fetch ---
Total: 144.62 M
```